src/services/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Class to VerifyErrors
+ */
+export default class VerifyErrors extends Error {
+  /**
+     * VerifyError constructor
+     * @param {number} status - Error Status
+     * @param {String} message - Error Message
+     */
+  constructor(status, message) {
+    super();
+    this.status = status;
+    this.message = message;
+  }
+}
```

Comment:
Please, I don't know why you abstracted this `VerifyErrors` as a class, remove it and follow the team's convention. Lets not over abstract our application. Also while abstracting, also remember you are working in a team. This class is not necessary, please remove it.